app/res/xml/preferences_developer.xml

```diff
@@ -6,8 +6,8 @@
         android:key="cc-superuser-enabled"
         android:title="Developer Mode Enabled"/>
     <ListPreference
```

Comment:
My current AS styling settings arranges the xml attributes alphabetically. Do we follow that practice or should I turn it off.